hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java

```diff
@@ -236,6 +238,30 @@ void testLDBScan(
     Assertions.assertTrue(stderr.toString().contains(stderrShouldContain));
   }
 
+  @Test
+  void testScanOfPipelinesWhenNoData() throws IOException {
+    // Prepare dummy table
+    prepareTable(PIPELINES, false);
+
+    // Prepare scan args
+    List<String> completeScanArgs = new ArrayList<>(Arrays.asList(
+        "--db", dbStore.getDbLocation().getAbsolutePath(),
+        "scan",
+        "--column-family", PIPELINES));
+
+    int exitCode = cmd.execute(completeScanArgs.toArray(new String[0]));
+    // Check exit code. Print stderr if not expected
+    Assertions.assertEquals(0, exitCode, stderr.toString());
+
+    if (exitCode == 0) {
```

Comment:
Nit: do we need this `if` after asserting the same?

Reply:
No need, it is a redundant. Thanks for noticing.